standup/status/models.py

```diff
@@ -12,7 +12,13 @@ class Project(models.Model):
     name = models.CharField(max_length=100, unique=True)
     slug = models.CharField(max_length=100, unique=True)
     color = models.CharField(max_length=6)
-    repo_url = models.URLField()
+    repo_url = models.URLField(max_length=100)
+
+    class Meta:
+        db_table = 'project'
+
+    def __str__(self):
+        return self.name
```

Comment:
I think `__str__` is for display in the admin, and `__repr__` in the REPL. I mostly added them when I first looked at the admin with the ported prod data and wanted to see more interesting names than "Project instance" or whatever.